SamplesDashboard/SamplesDashboard/ClientApp/src/types/samples.ts

```diff
@@ -9,8 +9,9 @@ export interface IRepositoryItem {
     pullRequests: any;
     issues: any;
     stargazers: any;
+    views: number;
     forks: any;
-    repositoryUrl: string;
+    url: string;
```

Comment:
There is no need to change the variable name

Reply:
The links weren't working initially because it couldn't tell what repositoryUrl was since its defined as url on the backend.